packages/expo-checkbox/src/ExpoCheckbox.tsx

```diff
@@ -1 +1,81 @@
+
+const ExpoCheckbox: CheckboxComponent = props => {
+  const { color, disabled, onChange, onValueChange, style, value, ...other } = props;
+
+  const handleChange = useCallback(() => {
+    onValueChange && onValueChange(!value);
+  }, [onValueChange, value]);
+
+  return (
+    <Pressable
+      {...other}
+      disabled={disabled}
+      // Announces "checked" status and "checkbox" as the focused element
+      accessibilityRole="checkbox"
+      accessibilityState={{ disabled, checked: value }}
+      style={[
+        styles.root,
+        style,
+        value && styles.checked,
+        !!color && { backgroundColor: value ? color : undefined, borderColor: color },
+        disabled && styles.disabled,
+        value && disabled && styles.checkedAndDisabled,
+      ]}
+      onPress={handleChange}>
+      {value && (
+        <Image
+          source={{
+            uri: checkmarkBase64,
+          }}
+          style={StyleSheet.absoluteFill}
+        />
+      )}
+    </Pressable>
+  );
+};
+
+ExpoCheckbox.displayName = 'Checkbox';
+
+export default ExpoCheckbox;
+
+export const name = 'ExpoCheckbox';
```

Comment:
what is this for?

Reply:
I have no idea, copied from the web component - will remove!